modules/_labs/color-picker/react/spec/ColorPicker.spec.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import {fireEvent, render} from '@testing-library/react';
+import {colors} from '@workday/canvas-kit-react-core';
+import {ColorPicker, ColorPickerProps} from '@workday/canvas-kit-labs-react-color-picker';
+
+describe('<ColorPicker /> component', () => {
+  const dataTestId = 'TESTY_TEST';
+
+  const renderColorPicker = (props?: Partial<ColorPickerProps>) => {
+    return render(<ColorPicker data-test-id={dataTestId} onChange={jest.fn()} {...props} />);
+  };
+
+  it('should include aria-label', () => {
+    const ariaLabel = 'Background Color';
+    const {container, unmount} = renderColorPicker({ariaLabel});
+
+    expect(container.querySelector(`[aria-label="${ariaLabel}"]`)).not.toBeNull();
+    unmount();
+  });
+
+  it('should call onChange handler when clicking a color swatch', () => {
+    const onChange = jest.fn();
+    const {container, unmount} = renderColorPicker({onChange});
+    const colorSwatch = container.querySelector('#canvas-color-picker--color-333333')!;
+
+    fireEvent.click(colorSwatch);
+    expect(onChange).toHaveBeenCalledWith(colors.blackPepper400);
+    unmount();
+  });
+
+  it('should call onChange handler when selecting a color swatch with ENTER', () => {
+    const onChange = jest.fn();
+    const {container, unmount} = renderColorPicker({onChange});
+    const colorSwatch = container.querySelector('#canvas-color-picker--color-333333')!;
+
+    fireEvent.mouseEnter(colorSwatch);
+    fireEvent.keyDown(colorSwatch, {key: 'Enter'});
```

Comment:
The `mouseEnter` isn't necessary here. This type of test should be covered by Cypress and not in the unit tests. If we use a `button` element, we don't have to do any special keyboard handling. `Enter` and `Space` keys already work on button elements.